Enhancement : Add back button in several screens.

dipu989 opened the pull request:
Fixes #445 

**Changes**:  Add back navigation button in several screens like 
1) DataLogger Screen
2) PinLayout Screen
3) Meditation Activity
4) Focus Activity etc

**Screenshot/s for the changes**: 
1) Mediation Screen

![Meditate](https://user-images.githubusercontent.com/31280303/64498287-800f6080-d2d1-11e9-923e-2e576b18d724.PNG)
2) Focus Screen

![focus](https://user-images.githubusercontent.com/31280303/64498289-843b7e00-d2d1-11e9-8c2d-428c3c3e281b.PNG)
3) Device Connector

![Connect](https://user-images.githubusercontent.com/31280303/64498295-88679b80-d2d1-11e9-8f8d-4e3fd734a5fa.PNG)
4) Meditation List

![Meditate1](https://user-images.githubusercontent.com/31280303/64498298-8a315f00-d2d1-11e9-88f3-eba37abb7a7f.PNG)
5) Meditation's Music Playing

![Meditate2](https://user-images.githubusercontent.com/31280303/64498299-8b628c00-d2d1-11e9-8ecc-2314735c0f63.PNG)

6) PinLayout

![PinLayout](https://user-images.githubusercontent.com/31280303/64498499-6589b700-d2d2-11e9-87db-e15e72038c26.PNG)







**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing** (Latest): 
[apkDebug.zip](https://github.com/fossasia/neurolab-android/files/3647861/apkDebug.zip)

jddeep commented:
@dipu989 Please recheck the apk you uploaded. It's not getting installed.

jddeep reviewed (changes requested):
@dipu989 Please pull the latest changes. A relax home screen was added recently. Please check its lifecycle and add the back buttons required there as well. Thank you.

jddeep reviewed (changes requested):
Resolve Conflicts. Squash commits into one. Give sensible commit name.

dipu989 commented:
@jddeep  After pulling latest changes from upstream, Development branch was updated with upstream. But the branches retains the old codes only and I have a PR open from an old branch. I need to make changes to the new code and add those changes to the old branch only. I made a new branch from updated development branch and then made changes! But I can not cherry pick the commits to the old branch now(The one having Open PR). I updated old branch by fetching and rebasing it with origin/master. But still no help! Can you help with this?

jddeep commented:
@dipu989 You should have pulled changes from upstream before making the PR. Please rebase to current upstream development. It seems you have pulled changes from an earlier stage of the development branch. Please learn git essentials and update this PR. Thank you.

dipu989 committed:
Add back nav button in several screens.

Feature : Add back button in several screens.

Feature : Add back button in several screens.

Removing extra line.

Removed extra line.

dipu989 commented:
@jddeep  I have made suitable changes and uploaded the latest apk. Please review!

jddeep commented on app/src/main/java/io/neurolab/activities/DataLoggerActivity.java:
```diff
@@ -9,6 +9,7 @@
 import android.support.v7.widget.RecyclerView;
 import android.view.View;
 import android.widget.TextView;
+
```
Remove

dipu989 replied:
@jddeep  Done with changes!

jddeep commented:
Nice...
